source/code/plugins/filter_syslog_security.rb

```diff
@@ -36,7 +36,7 @@ def filter(tag, time, record)
       # The time may be slightly in the future from the ingestion time.
       record['Timestamp'] = OMS::Common.format_time(Time.now.to_f)
 
-      record['Message'] = record['message']
+      record['Message'] = record['ident'] + ': ' + record['message']
       record.delete 'message'
       record.delete 'time'
```

Comment:
Can you explain why you need to do this? The 'ident' field is ingested into search as the 'ProcessName' field